### PR TITLE
fix: Update CLI for depcheck

### PR DESCRIPTION
**Pull request by jimhunty**
Adding depcheck to fix `yarn lint` on new packages.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/0b624bf36fe7ccfcf897eadfa6fa2d8d6ec25dc2/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b69ee57223c32c467c263aca5b1a35f2343ce147/index.html

File: lib/cli/package-json.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ module.exports = variables => ({
   main: `dist/${variables.packageName}`,
   dev: `src/${variables.packageName}`,
   scripts: {
+    depcheck:
+      "depcheck --ignores='babel-*,depcheck,eslint,@thetimes/jest-lint,identity-obj-proxy,jest,prettier,webpack*' --ignore-bin-package=false --skip-missing",
```
Comment (craigbilner) [inline lib/cli/package-json.js]: is `identity-obj-proxy` needed?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/6f10a019800b26668b8686a0e3b9eeeb08e49612/index.html

**Pull request merged**